frontend/src/lib/candidate/components/input/PhotoInput.svelte

```diff
@@ -9,6 +9,8 @@
 
   export let disabled: boolean = false;
   export let photo: Photo | undefined;
```

Comment:
I don't think this is meaningful, because at the moment photo is not part of the question-answer model. This means it does not really fit to the `InputFieldProps` as it does not need a question and the onChange is also handled differently. Maybe it can be thought of again in #580